sos/report/plugins/openstack_trove.py

```diff
@@ -45,9 +45,10 @@ def apply_regex_sub(self, regexp, subst):
 
     def postproc(self):
         protect_keys = [
-            "default_password_length", "notifier_queue_password",
-            "rabbit_password", "replication_password", "admin_password",
-            "dns_passkey", "transport_url", "memcache_secret_key"
+            ".*_(password|key)?",
```

Comment:
The `?` seems superfluous here. It will match e.g. `foo_` (though I dont expect such property/variable name).

Reply:
good catch